packages/nextjs/src/server/__tests__/__snapshots__/exports.test.ts.snap

```diff
@@ -7,6 +7,8 @@ exports[`/server public exports should not include a breaking change 1`] = `
   "API_VERSION",
   "AllowlistIdentifier",
   "AuthStatus",
+  "CLERK_JS_URL",
+  "CLERK_JS_VERSION",
```

Comment:
❓ why are those exported? are we sure about this change? Aren't those internals?
cc: @nikosdouvlis